bug_report_page.php

```diff
@@ -193,7 +193,14 @@
 $f_copy_notes_from_parent         = gpc_get_bool( 'copy_notes_from_parent', false );
 $f_copy_attachments_from_parent   = gpc_get_bool( 'copy_attachments_from_parent', false );
 
-$t_fields = config_get( 'bug_report_page_fields' );
+$t_user_access_level = user_get_access_level( auth_get_current_user_id(), $t_project_id );
+$t_access_levels = config_get( 'simple_report_level' );
+if ( in_array( $t_user_access_level, $t_access_levels ) ){
```

Comment:
The code should be simplified by using existing function `access_has_project_level`.

I didn't try, but the following line should be enough to replace the current line and the two lines above
`if( access_has_project_level( config_get( 'simple_report_level' ), $t_project_id ) ) {`

Furthermore this ensures that `$g_simple_report_level` could be set just to an integer treshold value like UPDATER, but also to an array.

Reply:
That line indeed works so will change this also